src/main/drivers/accgyro_fake.c

```diff
@@ -23,23 +23,6 @@
 #if defined(SIMULATOR_BUILD) && defined(SIMULATOR_MULTITHREAD)
 #include <stdio.h>
 #include <pthread.h>
-#define printf printf
-#define sprintf sprintf
```

Comment:
I see ...  `common/printf.h` should be fixed (possibly by  using system libraries instead) .. Redefining standard functions in non-standard way is bad idea in large project

Even using this define, gcc should emit warning about redefinition ..